R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
+#'
+#' Construct XML data structure for \code{modeltime.xml}.
```

Comment:
I personally like this split design--it increases the number of chunks, obviously, but the benefit is a lot of flexibility in how we mix-and-match data into the xml files.